app/src/main/java/com/codesoom/HttpEnum/HttpMethodCode.java

```diff
@@ -0,0 +1,21 @@
+package com.codesoom.HttpEnum;
+
+public enum HttpMethodCode {
```

Comment:
이런 `enum`에는 RFC 2616 문서의 링크를 같이 제공하면 좋겠죠?
```suggestion
/**
 * HTTP 프로토콜의 메소드를 정의합니다.
 * @link https://datatracker.ietf.org/doc/html/rfc2616#section-9
 */
public enum HttpMethodCode {
```